### DISCUSSION TITLE
How to insert image with relative path in markdown?

### DISCUSSION THREAD

**Discussion by geyao1995** [massCodeIO/massCode#200] [Q&A] [answered]
I set the relative path accroding to the `db.json` and it doesn't work. But the absolute path works.

**Comment by antonreshetov** [comment]
In this case, no way. The relative path in this context is the path to the filesystem root. ~~But you can't access the filesystem from the client side of the application because of [Context Isolation](https://www.electronjs.org/docs/latest/tutorial/context-isolation)~~

In this case through the absolute path you can, so it's just reading the file from the browser.

**Comment by antonreshetov** [comment, marked as answer]
I'll think about a solution. If it will generally be of interest to many people.

Can you tell me about the use cases?

**Reply by geyao1995** [reply]
I want to save some code snippets which can draw some figures (like [matplotlib](https://matplotlib.org/) code). So it is important to visualize the result figures the codes plot. I choose the markdown format to save codes and (local saved) figures at the same time. What worries me is that if I move the massCode to another device, the absolute path will prevent the figures from being displayed.

**Reply by antonreshetov** [reply]
The solution is simple, so expect in the next release 

![Запись экрана 2022-08-07 в 13 52 02](https://user-images.githubusercontent.com/17480987/183287327-8bc4517a-3def-45f3-9fd3-53e3077b987d.gif)

**Reply by geyao1995** [reply]
Thank you for the reply! I'm really excited for the next release.

**Reply by antonreshetov** [reply]
Just released [v3.1.0](https://github.com/massCodeIO/massCode/releases/tag/v3.1.0)
